contrib/cron-jobs/update-tor-nodes

```diff
@@ -1 +0,0 @@
-../../intelmq/bots/experts/tor_nodes/update-tor-nodes
```

Comment:
please leave it as is for backwards-compatibility

Reply:
Done.